### DISCUSSION TITLE
'eleve.id' remplacé par 'eleve.ine' ?

### DISCUSSION THREAD

**Discussion by ThierryM** [DSMejantel/Ecole_inclusive#10] [Ideas]
Adieu David,
Je suis en train d'adapter École Inclusive sur une seule école pour essayer de satisfaire une demande d'aide au suivi des aides pour les élèves. Sur cette école, il y a pas mal d'homonymes et le seul moyen de les différencier est l'utilisation de leur INE. Aussi au lieu d'utiliser comme clé primaire INTEGER 'eleve.id' pourquoi ne pas utiliser une clé primaire TEXT 'eleve.ine' qui correspondrait à l'INE de l'élève ? Forcément il n'y aurait pas d'incrémentation automatique : nécessité de saisir/importer cette donnée.
Cela permettrait aussi de réaliser des importations .csv de ONDE (application de gestion des élèves du 1er degré).

**Comment by DSMejantel** [comment]
C'est une idée à prendre en compte. L'INE est utile pour éviter des méprises. En revanche, sur SIECLE (équivalent d'ONDE pour le 2nd degré) les élèves arrivant sur l'académie ont parfois un INE inconnu lors de l'inscription.
Je maintiendrai les deux champs id (pour la raison invoquée et pour ne pas tout reécrire !) et INE. Je regarde cela de plus près demain...

**Reply by ThierryM** [reply]
Effectivement, j'ai remplacé eleve.id par eleve.ine et ça fait beaucoup de modifications... Mais ça me permet de rentrer dans le code ;-).
A lèu.
